nginx/nginx.go

```diff
@@ -111,6 +113,13 @@ func (c *Config) Build(params map[string]interface{}) ([]byte, error) {
 	return src, nil
 }
 
+func (c *Config) getBinary() string {
```

Comment:
the convention is this repo is to have a function called applyDefaults(). You can also put it in the beginning of Run().